recipe/meta.yaml

```
@@ -7,57 +7,81 @@ package:
 source:
   fn: mayavi-{{ version }}.tar.gz
   url: https://github.com/enthought/mayavi/archive/{{ version }}.tar.gz
-  sha256: c8addff9c87c7b9f91d32205c99d0af75cf480cbb2585e69149ab6b9337deb2e
+  sha256: eb7f46952d4786008ed4f272172bec475fcdcb988f5e1c5405daea9d41955cef
 
 build:
   number: 0
   entry_points:
     - mayavi2 = mayavi.scripts.mayavi2:main
     - tvtk_doc = tvtk.tools.tvtk_doc:main
-  skip: True  # [win and py27]
   ignore_run_exports:
     - pyqt    # [linux and aarch64]
+  # No vtk for osx-64
+  skip: true  # [osx and x86_64]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
     - {{ compiler('c') }}
-    - make    # [linux and aarch64]
   host:
     - python
-    - numpy
     - pip
     - setuptools
-    - vtk
-    - traitsui
-    - apptools
-    - sphinx  # [osx or (linux and aarch64)]
+    - numpy >=2.0.0rc2,<3
+    - vtk-base 9.4.1 
+    - pyqt {{ pyqt }}
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - vtk
-    - traitsui
-    - pyface
     - apptools
+    - configobj
     - envisage
-    - pyqt >=4,<6
-    - setuptools
+    - numpy
+    - pyface >=6.1.1
+    - pygments
+    - traits >=6.0.0
+    - traitsui >=7.0.0
+    - packaging
+    - importlib_resources
```

```suggestion
    - importlib_resources  # [py<311]
```
https://github.com/enthought/mayavi/blob/4.8.3/mayavi/__init__.py#L20